docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: '3'
+
+services:
+    assets:
```

Comment:
the order of definition of these containers is alphabetical, but order of their startup would help in visualizing the chain of dependencies `composer,npm`->`assets`->`cli,fpm`